packages/DataTable/src/components/Navigation/Navigation.js

```diff
@@ -4,12 +4,34 @@ import * as styled from "./Navigation.styles";
 
 const propTypes = {
   children: PropTypes.node.isRequired,
+  ColumnsDefinition: PropTypes.arrayOf(
+    PropTypes.shape({
+      props: PropTypes.object.isRequired,
+      type: PropTypes.func.isRequired,
+    })
+  ),
+};
+
+const defaultProps = {
+  ColumnsDefinition: [],
 };
 
 export default function Navigation(props) {
-  const { children } = props;
-  return <styled.Navigation>{children}</styled.Navigation>;
+  const { children, ColumnsDefinition } = props;
+  return (
+    <styled.Navigation>
+      {React.Children.map(children, child => {
```

Comment:
I think would be more useful to pass all the props to all component no only to the Sort.
```
<styled.Navigation>
  {React.Children.map(Children, child => React.CloneElement(child, {...props,  ...child.props)) 
</styled.Navigation>
```

Reply:
Ok! I was unsure about what kind of child components in the future, but it makes sense to keep it simple and clear